packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -10,20 +10,20 @@ import {
   SidebarScrollWrapper,
 } from './common';
 import AlertContext from '../../contexts/AlertContext';
-import SidebarUserButton from './SidebarUserButton';
 import useHideMobileSidebar from '../../hooks/useHideMobileSidebar';
-import MyFeedButton from './MyFeedButton';
-import { SidebarBottomSectionSection } from './SidebarBottomSection';
-import { DiscoverSection } from './DiscoverSection';
-import { ContributeSection } from './ContributeSection';
-import { ManageSection } from './ManageSection';
-import { MobileMenuIcon } from './MobileMenuIcon';
 import AuthContext from '../../contexts/AuthContext';
 import { getFeedName } from '../MainFeedLayout';
-import { SquadsList } from './SquadsList';
```

Comment:
Remove the `SquadsList.tsx` file as well?

Reply:
Ah yes forgot about that one ☝️